plugins/modules/notification/cisco_webex.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: cisco_webex
+short_description: Send a webexmsg to a Cisco Webex Teams Room or Individual.
```

Comment:
```suggestion
short_description: Send a webexmsg to a Cisco Webex Teams Room or Individual
```
"Unlike description:, short_description should not have a trailing period/full stop." from the [doc](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block)